### DISCUSSION TITLE
Blobs/zits in print from print pausing.

### DISCUSSION THREAD

**Discussion by Ketzukhan** [feelfreelinux/octo4a#80] [Q&A]
Blobs/zits accumulating during a pause mid print. Seems like what other people are having with the standard octoprint when they use the Raspberry Pi and have to go into the. H files and change their configs. I tried to look for the same files in the file browser for this Android and can't find the concrete file to where I can change the settings. Ex: buffer blocker etc. Anyone else having this issue. Same exact issue in the YouTube video linked below...
https://youtu.be/rG8C9jSSZbI

**Comment by feelfreelinux** [comment]
Hi! Can you send me which files people edit to fix it in normal octoprint?

**Reply by sbaeder** [reply]
That video is talking about using something to eliminate the MANY short moves, since that is a lot of data to send to the computer.  It is mostly talking about changing printer firmware to increase the baud rate and internal printer buffer.  The cause of the problem is that too many short move commands that don't take long to print, but because of a slow transmission speed (baud rate), OctoPrint can't send the actual g-code for the moves fast enough.  This causes the printer to wait until the full command (all the text for the complete move command) is sent to it.  

If it can send faster, you don't see this issue.  So, this is just a BAUD setting, and that is configured in Octoprint BUT it also has to match the printers Baud rate (in the printer firmware.

The video also mentions ArcWelder (which came out after the video).  This plugin analyzes all the short the moves in the gcode file and if possible, changes them to ONE arc command (that takes long enough for the sender (OctoPrint) to keep the printer buffer from becoming empty

So, the best "fix" is to use ArcWelder (if you can).  Otherwise, you need to edit/configure/FLASH new firmware to increase Baud rate...
